python/tests/grades.csv

```diff
@@ -0,0 +1,101 @@
+student_id,science,maths,art
```

Comment:
It would be useful to add in the test documentation the seeded random data generation code that generates the data here. Also the names in my opinion should just be "Firstname1 Lastname1", "Firstname2 Lastname2" etc. rather than auto-generated realistic ones.

Reply:
This data is from a random data generator website, but yeah, it'll likely be better to use a generator we can control.